release/merge.sh

```diff
@@ -0,0 +1,44 @@
+#
```

Comment:
Maybe include the hash bang? Does the particular shell matter?

Reply:
The hashbangs are only required if you execute the scripts directly, e.g.

    $ ./merge.sh

In this case, it's impossible, because the scripts do not have execute permissions. This is by design, so that you have to explicitly specify the shell and do:

    $ bash merge.sh

In practice, shells other than bash may be usable, but I haven't tested them.